src/main/java/com/bandwidth/sdk/messaging/MessagingClient.java

```diff
@@ -76,4 +87,109 @@ public Message sendMessage(SendMessageRequest request) throws ExecutionException
         }
     }
 
+    /**
+     * Uploads MMS media content. Max 65MB
```

Comment:
While it's true the media endpoint can support 65MB, an MMS won't be able to send this much. This probably confuses the customer more than it helps